spec/services/application_choices_export_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
```

Comment:
How is this file part of the diff? It's already on master 🤔

Reply:
I had to rename it from `spec/services/applications_export_spec.rb` to `spec/services/application_choices_export_spec.rb` in order to get a clean Rubocop run